theming/theme-package.rst

```diff
@@ -536,14 +532,15 @@ Replace the placeholder with the real Plone top-navigation links:
      css:theme-children=".navbar-nav"
      css:content-children=".plone-navbar-nav" />
 
-Here we take the list of links from Plone and replace the placeholder links in the theme with it.
+Here we take the list of links from Plone and replace the placeholder links in
+the theme with it.
```

Comment:
I actually agree but hadn't seen that so thanks! Most lines were already with 78/79/80 chars so I just followed that and fixed exceptions. I will do like this from now on, is that ok? (These docs are supposed to be completely rewritten still this year)